detection_rules/devtools.py

```diff
@@ -635,6 +635,32 @@ def license_check(ctx, ignore_directory):
     ctx.exit(int(failed))
 
 
+@dev_group.command('test-version-lock')
+@click.argument('branches', nargs=-1, required=True)
+@click.option('--remote', '-r', default='origin', help='Override the remote from "origin"')
+def test_version_lock(branches: tuple, remote: str):
```

Comment:
We potentially could run this in all unit tests where a rule file is changed to be really thorough, but I think for now we should leave it as a manual command